### DISCUSSION TITLE
Approach circle disappearing does not match stable

### DISCUSSION THREAD

**Discussion by Kuuuube** [ppy/osu#37786] [Ideas]
### Type

Cosmetic

### Bug description

The way the approach circle is removed is slightly different from stable. In stable the approach circle disappears instantly at the end of its life. In lazer it quickly fades out instead.

I've provided a side by side comparison below (0.067x speed). The skin used can be downloaded here https://kuuube.s-ul.eu/5ExHepfX.

### Screenshots or videos

![stable_vs_lazer_approach_circle_end](https://github.com/ppy/osu/assets/61125188/2e4f3cf6-52aa-407c-b4d2-a03bb716c8d5)

### Version

2023.1221.0-lazer

### Logs

N/A

**Comment by peppy** [comment]
Pretty sure this is intentional, when the object is not clicked it will remain longer.

**Comment by Pallid3** [comment]
I also noticed the issue today and it looks ugly with certain skins. Maybe reconsider matching it with stable?
